nautobot/ipam/querysets.py

```diff
@@ -191,46 +192,50 @@ def string_search(self, search):
 class PrefixQuerySet(BaseNetworkQuerySet):
     """Queryset for `Prefix` objects."""
 
-    def net_equals(self, prefix):
-        prefix = netaddr.IPNetwork(prefix)
-        last_ip = self._get_last_ip(prefix)
-        return self.filter(prefix_length=prefix.prefixlen, network=prefix.network, broadcast=last_ip)
-
-    def net_contained(self, prefix):
-        prefix = netaddr.IPNetwork(prefix)
-        last_ip = self._get_last_ip(prefix)
-        return self.filter(
-            prefix_length__gt=prefix.prefixlen,
-            network__gte=prefix.network,
-            broadcast__lte=last_ip,
-        )
-
-    def net_contained_or_equal(self, prefix):
-        prefix = netaddr.IPNetwork(prefix)
-        last_ip = self._get_last_ip(prefix)
-        return self.filter(
-            prefix_length__gte=prefix.prefixlen,
-            network__gte=prefix.network,
-            broadcast__lte=last_ip,
-        )
-
-    def net_contains(self, prefix):
-        prefix = netaddr.IPNetwork(prefix)
-        last_ip = self._get_last_ip(prefix)
-        return self.filter(
-            prefix_length__lt=prefix.prefixlen,
-            network__lte=prefix.network,
-            broadcast__gte=last_ip,
-        )
-
-    def net_contains_or_equals(self, prefix):
-        prefix = netaddr.IPNetwork(prefix)
-        last_ip = self._get_last_ip(prefix)
-        return self.filter(
-            prefix_length__lte=prefix.prefixlen,
-            network__lte=prefix.network,
-            broadcast__gte=last_ip,
-        )
+    def net_equals(self, *prefixes):
+        query = Q()
+        for prefix in prefixes:
+            with contextlib.suppress(netaddr.AddrFormatError, ValueError):
+                prefix = netaddr.IPNetwork(prefix)
+                last_ip = self._get_last_ip(prefix)
+                query |= Q(prefix_length=prefix.prefixlen, network=prefix.network, broadcast=last_ip)
+        return self.filter(query)
```

Comment:
Did this change the previous behavior where if any input is invalid we return `queryset.none()`? It looks like now if there's an exception raised we just skip that prefix and fail "open" instead?